README.md

```diff
@@ -73,6 +73,8 @@ There are a couple out-of-band access methods you can use to explore Bottlerocke
 Either option will give you a shell within Bottlerocket.
 From there, you can [change settings](#settings), manually [update Bottlerocket](#updates), debug problems, and generally explore.
 
+**Note:** These methods require that your instance has permission to access the ECR repository where these containers live; the appropriate policy to add to your instance's IAM role is `AmazonEC2ContainerRegistryReadOnly`.
```

Comment:
This is a good general start, but allows the instance to pull from any registry hosted in ECR.  We may want to additionally document the registry that contains our Bottlerocket images in case customers wish to define a more tightly-scoped set of permissions.

Reply:
Opened #857 to track